src/planet-4-151612/openresty/bin/nginx_configure_geoip2.sh

```diff
@@ -1,29 +1,58 @@
 #!/usr/bin/env bash
 set -euo pipefail
+#Check License and Account key variables
 
-_good "$(printf "%-10s " "openresty:")" "GEOIP2 ${GEOIP2_ENABLED}"
-
-[[ ${GEOIP2_ENABLED} = "true" ]] || {
+if [[ ${PHP_ENABLED} = 'true' ]]
```

Comment:
Please simplify exit logic, keep conditionals at single depth.

Test for GEOIP_ENABLED || exit quietly
Test for PHP_ENABLED && exit with failure
Test License data || exit failure

... proceed with script